gapic-generator/bin/gapic-generator

```diff
@@ -30,10 +53,10 @@ OptionParser.new do |opts|
     options[:export] = export_path
   end
 
-  # opts.on "-g", "--gem name", String,
-  #         "Generate a new rubygem" do |gem_name|
-  #   options[:gem] = gem_name
-  # end
+  opts.on "-g", "--gem name, path", Array, "Generate a new rubygem" do |args|
```

Comment:
Can the `name` and `path` inputs each have their own option? If not, remove the space, since it doesn't work in practice: `--gem name,path`